app/models/digest_run_subscriber.rb

```diff
@@ -0,0 +1,11 @@
+class DigestRunSubscriber < ApplicationRecord
+  validates :digest_run_id, :subscriber_id, presence: true
+  belongs_to :digest_run
+  belongs_to :subscriber
+
+  scope :incomplete_for_run, ->(digest_run_id) { where(digest_run_id: digest_run_id).where(completed_at: nil) }
+
+  def mark_completed!
```

Comment:
This is called `mark_completed!` but we called the one on `DigestRun` `mark_complete!`, I reckon it would be good to call them the same. I don't mind which though.